src/actions/exchangeActions.js

```diff
@@ -316,7 +316,7 @@ export const markNotificationAsSeenAction = () => {
   };
 };
 
-export const getExchangeSupportedAssetsAction = () => {
+export const getExchangeSupportedAssetsAction = (callback?: () => void) => {
```

Comment:
Do we really need callback here if I can't see any calls with callback param?

Reply:
We're calling it in Exchange screen's componentDidMount